examples/sandbox/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hdwallet-sandbox",
-  "version": "1.50.4",
+  "version": "1.50.5-alpha.68",
```

Comment:
made the mistake of bumping package to non alpha in the metamask-snaps, but if we can get an update version whitelisted, would be nice to make this a non alpha release now that we are live I feel like.